test/test_transforms.py

```diff
@@ -1997,5 +1757,244 @@ def test_random_order():
     random_order_transform.__repr__()
 
 
+def test_five_crop():
+    to_pil_image = transforms.ToPILImage()
+    h = random.randint(5, 25)
+    w = random.randint(5, 25)
+    for single_dim in [True, False]:
+        crop_h = random.randint(1, h)
+        crop_w = random.randint(1, w)
+        if single_dim:
+            crop_h = min(crop_h, crop_w)
+            crop_w = crop_h
+            transform = transforms.FiveCrop(crop_h)
+        else:
+            transform = transforms.FiveCrop((crop_h, crop_w))
+
+            img = torch.FloatTensor(3, h, w).uniform_()
+            results = transform(to_pil_image(img))
+
+            assert len(results) == 5
+            for crop in results:
+                assert crop.size == (crop_w, crop_h)
+
+            to_pil_image = transforms.ToPILImage()
+            tl = to_pil_image(img[:, 0:crop_h, 0:crop_w])
+            tr = to_pil_image(img[:, 0:crop_h, w - crop_w:])
+            bl = to_pil_image(img[:, h - crop_h:, 0:crop_w])
+            br = to_pil_image(img[:, h - crop_h:, w - crop_w:])
+            center = transforms.CenterCrop((crop_h, crop_w))(to_pil_image(img))
+            expected_output = (tl, tr, bl, br, center)
+            assert results == expected_output
+
+
+@pytest.mark.parametrize(should_vflip, [True, False])
+@pytest.mark.parametrize(single_dim, [True, False])
+def test_ten_crop(should_vflip,single_dim):
```

Comment:
This should be:


```suggestion
@pytest.mark.parametrize('should_vflip', [True, False])
@pytest.mark.parametrize('single_dim', [True, False])
def test_ten_crop(should_vflip, single_dim):
```


Please make sure to read the instructions at https://github.com/pytorch/vision/issues/3945#issue-909493806 and you can take inspiration from https://github.com/pytorch/vision/pull/3907/files to see how to parametrize tests.

Also I really recommend to try your changes locally before you push: it will be much easier for you to see what works and what doesn't work